packages/gallery/src/components/gallery/proGallery/slideshowView.js

```diff
@@ -460,6 +463,11 @@ class SlideshowView extends GalleryComponent {
         e.preventDefault();
```

Comment:
I see two problems in this code:

1. This entire logic is duplicated. The only difference between these two `case`s is the `direction`. 
2. There's too much logic here that can be extracted (and tested) outside of the class (no `this` or `bind`).


[This commit](https://github.com/wix/pro-gallery/commit/e828430885db46f41d676a0c014e0807ddb7dcc9#diff-521cda42545179515a78af227d78e504a0391b727945c20b1aa94afa1f621156R464-R472) has the two above issues fixed

Reply:
Thanks👍